error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T12:50:52.006Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T12:53:40.804Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:00:08.413Z"}
error: 
    status - 500 
    message - ValidationError: "emailid" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:00:15.854Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:00:20.403Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:01:12.205Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:04:32.736Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:04:49.719Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:07:37.935Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:11:23.175Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:12:28.346Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:12:36.899Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:13:33.062Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:13:49.088Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:14:38.545Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:15:50.873Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:16:09.777Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:17:20.488Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:17:32.622Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:18:02.238Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:18:18.016Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:19:15.227Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:19:24.845Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:20:24.536Z"}
error: Could not connect to the database. Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/ {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"ReplicaSetNoPrimary"},"stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\project1_googleapi\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\project1_googleapi\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:9)\n    at D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:27:7\n    at new Promise (<anonymous>)\n    at apply (D:\\project1_googleapi\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:19:12)\n    at database (D:\\project1_googleapi\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\project1_googleapi\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Module._compile (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Object.newLoader [as .js] (D:\\project1_googleapi\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.runMain (node:internal/modules/run_main:135:12)\n    at Object.<anonymous> (D:\\project1_googleapi\\node_modules\\@babel\\node\\src\\_babel-node.ts:222:12)\n    at Module._compile (node:internal/modules/cjs/loader:1376:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1435:10)\n    at Module.load (node:internal/modules/cjs/loader:1207:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1023:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-01-25T13:20:27.661Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.insertOne()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\project1_googleapi\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:573:17)
    at processTimers (node:internal/timers:514:7) 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:21:16.649Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-25T13:26:31.862Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/newuser1 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-27T09:29:44.391Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee5$ (D:\project1_googleapi\src\services\/user.service.js:37:47)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getUser (D:\project1_googleapi\src\services\/user.service.js:36:21)
    at _callee2$ (D:\project1_googleapi\src\controllers\/user.controller.js:31:36)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at getUser (D:\project1_googleapi\src\controllers\/user.controller.js:29:21)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:365:14)
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:421:3)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at jsonParser (D:\project1_googleapi\node_modules\body-parser\lib\types\json.js:110:7)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at urlencodedParser (D:\project1_googleapi\node_modules\body-parser\lib\types\urlencoded.js:91:7) 
    url - /api/v1/newuser1/65b4cd0d1c94330f8c97cec6 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-27T09:30:01.496Z"}
error: 
    status - 500 
    message - ReferenceError: email is not defined
    at _callee5$ (D:\project1_googleapi\src\services\/user.service.js:37:47)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at Object.apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at Object.getUser (D:\project1_googleapi\src\services\/user.service.js:36:21)
    at _callee2$ (D:\project1_googleapi\src\controllers\/user.controller.js:31:36)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at getUser (D:\project1_googleapi\src\controllers\/user.controller.js:29:21)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:365:14)
    at param (D:\project1_googleapi\node_modules\express\lib\router\index.js:376:14)
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:421:3)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12) 
    url - /api/v1/newuser1/65b4cd0d1c94330f8c97cec6 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-27T09:31:49.815Z"}
error: 
    status - 500 
    message - ValidationError: "first_name" is required 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:11:43.683Z"}
error: 
    status - 500 
    message - ValidationError: "lastName" is not allowed 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:12:06.450Z"}
error: 
    status - 500 
    message - ValidationError: "email_id" is required 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:18:38.450Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'json')
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:31:9)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:18:56.281Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:18:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:26:28.555Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:27:30.361Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:38:40.044Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T18:45:23.751Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:11:13.109Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:16:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:11:20.731Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:17:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:15:43.666Z"}
error: 
    status - 500 
    message - Error: User with the same email already exists
    at _callee2$ (D:\project1_googleapi\src\services\/user.service.js:17:11)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-28T19:22:06.002Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/user/registration 
    method - POST 
    IP - ::1
   {"timestamp":"2024-01-31T07:12:59.359Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:15:53.620Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:16:50.483Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:16:58.508Z"}
error: 
    status - 500 
    message - JsonWebTokenError: jwt must be provided
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:53:17)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:17:09.350Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:17:22.906Z"}
error: 
    status - 500 
    message - JsonWebTokenError: invalid signature
    at D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:133:19
    at getSecret (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:90:14)
    at Object.verify (D:\project1_googleapi\node_modules\jsonwebtoken\verify.js:94:10)
    at _callee$ (D:\project1_googleapi\src\middlewares\/auth.middleware.js:22:32)
    at tryCatch (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:27:7
    at new Promise (<anonymous>)
    at apply (D:\project1_googleapi\node_modules\@babel\runtime\helpers\asyncToGenerator.js:19:12)
    at userAuth (D:\project1_googleapi\src\middlewares\/auth.middleware.js:12:22)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\project1_googleapi\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (D:\project1_googleapi\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\project1_googleapi\node_modules\express\lib\router\index.js:175:3)
    at router (D:\project1_googleapi\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\project1_googleapi\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\project1_googleapi\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\project1_googleapi\node_modules\express\lib\router\index.js:328:13)
    at D:\project1_googleapi\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\project1_googleapi\node_modules\express\lib\router\index.js:346:12)
    at next (D:\project1_googleapi\node_modules\express\lib\router\index.js:280:10)
    at D:\project1_googleapi\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:206:9)
    at invokeCallback (D:\project1_googleapi\node_modules\raw-body\index.js:231:16)
    at done (D:\project1_googleapi\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (D:\project1_googleapi\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:518:28)
    at IncomingMessage.emit (node:domain:488:12)
    at endReadableNT (node:internal/streams/readable:1696:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/user/profile 
    method - GET 
    IP - ::1
   {"timestamp":"2024-01-31T07:17:33.344Z"}
